public/wp-content/themes/ColoredCow/src/scss/style.scss

```diff
@@ -33,5 +33,26 @@ Text Domain: coloredcow
 		display: block;
 	}
 }
+body{
+	padding-top: 150px;
+	background-image: url('RSVP_IMAGES/bg.png');
```

Comment:
folder names should be in small. Use dist/img folder for all theme specific images.